### PR TITLE
Translate Layout section

### LINKED ISSUE #6: format for nested bullet list is not right in English version of guide.
Check: https://github.com/ppresume/docs-notes/blob/22a55c87360da232df5c05d81dc2b147c769c216/guide-en.org

image

Hint: 
- https://en.wikipedia.org/wiki/Non-breaking_space

### PR DESCRIPTION
**Pull request by tarvos21**
Translate Layout section: introduction, font size, leading, page layout

Also: fix #6, the non-breaking space problem

### CODE REVIEW

File: guide-en.org [hunk 1]
```diff
@@ -198,57 +198,25 @@ In addition, there are situations where two or three em-dashes are joined togeth
...
+Typesetting is like building a two-dimensional architecture.
 
-If the text and its font is the building material, then the layout design is the architectural drawings. Before discussing the layout design, we need to first clear some of the basic concepts, especially in the layout of the commonly used unit of size system.
+If the text and its fonts are the materials for construction, then the layout design is the architectural drawing. Before discussing the design of layout, we need to make some of the basic concepts clear, especially the unit system of font size in typography.
 
-** font size
+** Font size
 
-Size (size) is a distinction between the size of the text of a measurement standard, the international common point system in China is based on the point system, supplemented by number system.
+Font size is the standard measure of how large the characters in text are. The international universal unit of font size is the point.
```
Comment (xiaohanyu) [inline guide-en.org]: Typo: "text are" -> "type area".

File: guide-en.org [hunk 2]
```diff
@@ -218,39 +218,26 @@ Most fonts will make the best effects when set to 10-12 pt. In other words, both
...
-- Chinese and Western mixed, the proportion of the use of Western fonts, Arabic numerals can use the proportion of fonts or monospaced fonts
-- Chinese characters and Western characters, the number between the proposal to add a certain margin, you can set the software to achieve high-level features through layout, you can manually add a half-size space
+Layout is the outline of typesetting, it's like the head-rope of a fishing net, once the head-rope is raised up, all the meshes spread naturally. In addition to general typesetting rules for font size and leading, there are some unique requirements in resume layout design. In principle, it's best to control the resume in one page.
+- Making the language clear and concise
+- Adjusting the font size and leading within a reasonable range
+- Keeping page margins in 10-25 mm and making the left and right margins symmetrical
+- Using lists rather than parallel sentences to list information
+- Using flush left alignment, taking caution to use justified alignment
+  - Justified alignment may cause problems in hyphenation and letter-spacing because the lengths of words are different
```
Comment (xiaohanyu) [inline guide-en.org]: I'm thinking about whether we should use Capitalized words for bullet list. However, right now, no definite answer. So just mention this, and I think we should propose a final big PR to fix consistency issue.

File: guide-en.org (hunk 1 above)
Comment (tarvos21) [inline guide-en.org]: Oh, the structure seems easy to misunderstand, `how large the characters in text are` is a clause modified from `how large are the characters in text`, as the objective of `of`.
I will change it in this way: 
`Font size is the standard measure of the characters's size in text.`
Will this be OK?

Comment (xiaohanyu) [inline guide-en.org]: Oh, seems I've made a stupid mistake about the English grammar. 

Yeah, prefer "Font size is the standard measure of the characters's size in text" for its brevity. 

Many thanks!

**Pull request merged**